pygmt/tests/test_coast.py

```diff
@@ -72,3 +73,35 @@ def test_coast_dcw_list():
         dcw=["ES+gbisque+pgreen", "IT+gcyan+pblue"],
     )
     return fig
+
+
+@pytest.mark.mpl_image_compare
+def test_coast_clip_land():
+    """
+    Test to clip dry areas.
+    """
+    region = [-28, -10, 62, 68]
+    grid = load_earth_relief(resolution="10m", region=region)
+
+    fig = Figure()
+    fig.basemap(region=region, projection="M12c", frame=True)
+    fig.coast(resolution="l", clip="land")
+    fig.grdimage(grid=grid, shading=True, cmap="batlow")
+    fig.coast(clip="end")
+    return fig
+
+
+@pytest.mark.mpl_image_compare
+def test_coast_clip_water():
+    """
+    Test to clip wet areas.
+    """
+    region = [-28, -10, 62, 68]
+    grid = load_earth_relief(resolution="10m", region=region)
+
+    fig = Figure()
+    fig.basemap(region=region, projection="M12c", frame=True)
+    fig.coast(resolution="l", clip="water")
+    fig.grdimage(grid=grid, shading=True, cmap="batlow")
```

Comment:
```suggestion
    fig.grdimage(grid=grid, cmap="batlow")
```

Reply:
I do like the Scientific Colour Maps, but because those colourmaps have a tendency to get updated with minor tweaks once in a while, maybe use one of the standard GMT CPTs at https://docs.generic-mapping-tools.org/6.3/cookbook/cpts.html#id2 that doesn't change (e.g. 'relief') so we don't need to update this test when a new SCM version lands.